tests/testthat/test-set_analysis_inputs_path.R

```diff
@@ -0,0 +1,57 @@
+test_that("sets appropriate global variables", {
+  data_location_ext <- "xxx"
+
+  output <-
+    callr::r(
+      func = function(set_analysis_inputs_path, data_location_ext) {
+        set_analysis_inputs_path(data_location_ext)
+      },
+      args = list(set_analysis_inputs_path, data_location_ext)
+    )
+
+  expect_equal(output, data_location_ext)
+})
+
+test_that("sets appropriate global variables", {
```

Comment:
can you be more specific with the name of this test? I am trying to figure out what it is expecting to happen here

Reply:
```suggestion
test_that("returns expected value", {
```
These are wrong and were carried over from another test. Is this better?

Reply:
or better... 
```suggestion
test_that("returns expected value when overridden by a global variable `proj_data_location_ext`", {
```